modules/comment-likes/comment-like-count.js

```diff
@@ -0,0 +1,65 @@
+var jetpackCommentLikeCount;
+
+( function( $ ) {
+	jetpackCommentLikeCount = {
+		jsonAPIbase: 'https://public-api.wordpress.com/rest/v1',
+		APIqueue: [],
+
+		getCommentLikeCounts: function() {
+			$( '.comment-like-count' ).each( function() {
+				var blogId = $( this ).attr( 'data-blog-id' );
+				var commentId = $( this ).attr( 'data-comment-id' );
+
+				jetpackCommentLikeCount.APIqueue.push( '/sites/' + blogId + '/comments/' + commentId + '/likes' );
+			} );
+
+			jetpackCommentLikeCount.fetchCounts();
+		},
+
+		showCount: function( commentId, count ) {
+			if ( count > 0 ) {
+				$( '#comment-like-count-' + commentId ).find( '.like-count' ).hide();
+				$( '#comment-like-count-' + commentId ).find( '.like-count' ).text( count );
+				$( '#comment-like-count-' + commentId ).find( '.like-count' ).fadeIn();
+			}
+		},
+
+		fetchCounts: function() {
+			var batchRequest = {
+				path:    '/batch',
+				data:    'urls[]=' + jetpackCommentLikeCount.APIqueue.join( '&urls[]=' ),
+				success: function( response ) {
+					for ( var path in response ) {
+						if ( ! response[ path ].error_data ) {
+							var urlPieces = path.split( '/' );
+							var commentId = urlPieces[ 4 ];
+							jetpackCommentLikeCount.showCount( commentId, response[ path ].found );
+						}
+					}
+				},
+				error: function() {}
+			};
+
+			jetpackCommentLikeCount.request( batchRequest );
+		},
+
+		request: function( options ) {
+			return $.ajax( {
+				type: 'GET',
+				url: jetpackCommentLikeCount.jsonAPIbase + options.path,
+				dataType : 'jsonp',
+				data: options.data,
+				success: function( response ) {
+					options.success( response );
+				},
+				error: function( response ) {
+					options.error( response );
+				}
+			} );
+		}
+	};
+} )( jQuery );
```

Comment:
I've removed the global variable in 9aaef66b8485d7aa1eab7ec9eba6d4648ae72d24, and simplified the remaining code. 

> also odd was seeing `this` inside the IIFE, which I don't think I normally see.

That `this` is not related to the outside scope, it's only relevant for `jQuery( '.comment-like-count' ).each`.